packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -100,12 +104,17 @@ const MT5AccountCard = ({
                             {Object.keys(specs).map((spec_attribute, idx) => (
                                 <tr key={idx} className='mt5-account-card__specs-table-row'>
                                     <td className='mt5-account-card__specs-table-attribute'>
-                                        <p className='mt5-account-card--paragraph'>{spec_attribute}</p>
+                                        <Text as='p' className='mt5-account-card--paragraph'>
+                                            {spec_attribute}
+                                        </Text>
                                     </td>
                                     <td className='mt5-account-card__specs-table-data'>
-                                        <p className='mt5-account-card--paragraph mt5-account-card--paragraph-right'>
+                                        <Text
+                                            as='p'
+                                            className='mt5-account-card--paragraph mt5-account-card--paragraph-right'
```

Comment:
Same here.

Reply:
I think it's best to follow “separation of concerns” concept just to keep the code tidy and neat, and since we have the respective scss file for the component, it is better to let the scss handle the conditional css rendering for the alignments.